### DISCUSSION TITLE
High-level goals, use cases, and priorities

### DISCUSSION THREAD

**Discussion by keyserj** [amelioro/ameliorate#547] [General]
Ameliorate's ultimate goals can be found here: [Mission & Vision](https://ameliorate.app/docs#mission--vision). There are a lot of possible use cases that fit within these goals. I'd like the tool to support all of these use cases well (if possible), but there's a lot of work to do, so some must be prioritized.

Thus far, I've avoided building features that don't benefit all/most of the use cases. So most of what I've built consists of improving the information structure (problem breakdown + auxiliary details) and UX for managing that. It seems to me that these still need to be further improved (particularly the UX), since they're not easily understood & picked up by new users.

Once new users are able to understand the structure and UX, I think features for mobile, collaborative, and long-term use cases would be next (maybe prioritized in that order), since they seem like they could help get people interested and engaged with the tool.

### Possible use cases
Below is the list of variables I use for thinking about all the possible use cases.

*denotes the ones I've been focusing on (i.e. they don't require special features, features for them seem to benefit all/most other use cases)

- number of people involved with a topic
	- *individual - i.e. use tool to organize thoughts
	- *small (2-8)
	- medium (9-30?)
	- large (31+)
- expected life of a topic
	- *short-term (e.g. use for a couple weeks, make a decision, no need to return to it)
	- long-term (e.g. make a topic & plan to improve it over months or years)
- expected intention of people contributing
	- *good faith
	- bad faith
- how invested people are in a topic
	- *invested
	- not very invested
- topic intended to gain public involvement
	- yes - it's desirable to share/get more people involved
	- *no - everyone that's needed is already known
- canonical
	- yes - intended to be an official topic, thorough, not duplicate with others
	- *no - perhaps want to keep to a small group that know the context, rather than needing to detail the topic to be "context-free"
- problem specificity
	- general (applies in many contexts e.g. world hunger)
	- *specific (applies to one context e.g. hunger in region X)
- problem complexity
	- "wicked" (e.g. social problems, affect a lot of people differently, tied with a lot of other problems)
	- high
	- *medium
	- *low (e.g. cars going too fast in neighborhood)
- shape of topic
	- *high-detail problem, low-detail/no solutions (e.g. for understanding a problem)
	- *low-detail/no problems, high-detail solution(s) (e.g. proposal, solves multiple things without targeting a specific problem)
	- *high-detail problem, high-detail solutions
- sync vs async
	- *sync - e.g. app is being used during a meeting, probably screenshared
	- *async - e.g. app is being used without live communication between contributors
- desktop vs mobile
	- *desktop
	- mobile

**Comment by keyserj** [comment]
@prototyperspective after reading some of your posts, I thought I'd clarify what my priorities have been thus far, and open up a discussion about it. Let me know if you have any thoughts.

**Reply by prototyperspective** [reply]
Interesting, I think the UX is not only for new users – I think it's more the UI in general since one would want to use this tool to make sense of or oversee something complex where the usefulness would be in reducing the information overload and info dispersion etc that currently can't be done. I think ultimately the value and need for this is mainly, not in Q&A stackexchange-like short answers about what to do for a quite specific problem, but in general problems of large complexity that need such a tool to be addressed. It may something most missing in societal problem-solving which it could boost, e.g. by enabling the missing pieces or bottleneck problems to get innovation or funding or thought etc & broadly enabling applying collective intelligence to problems.

**Reply by keyserj** [reply]
> the usefulness would be in reducing the information overload and info dispersion etc that currently can't be done

I think manual filtering + Quick Views already help some cases here, at least for experienced users, so I was mainly thinking of this as necessary for newer users. But I agree that even as an experienced user, there are plenty of cases that feel like manual filtering is not enough, and other UX features/decluttering could help significantly with those.

> I think ultimately the value and need for this is mainly [...] in general problems of large complexity

I think this can help with specific/smaller problems of some complexity too, but that as complexity goes up and problems are harder to understand/manage, that existing tooling is less and less sufficient.

**Comment by prototyperspective** [comment]
There also is another approach when it comes to high-level goals and use-uses besides such variables. That is particular use-cases. I think a key one would be [evidence-based effective **policy**](https://en.wikipedia.org/wiki/Evidence-based_policy) as a system for merging and integrating opposing views and crowdsource / collective intelligence. Policymakers could use to make policy that doesn't have large unmitigated problems (like high costs, economic downsides, or bureaucracy-issues) and civic society could use to identify what they propose or call for. It can also be used by scientists to identify bottleneck issues or broadly important nodes with too little data/research to work on. I think broadly the main use case would be societal problem-solving; there are other sites like stackexchange sites and reddit for smaller scale and/or somewhat-personal problems. Often problems need holistic and/or structural solutions where everything is tied to everything somehow so organizing complex things in these (possibly linked) maps could be quite useful for that. Also people are frustrated with politics or often can't communicate why they support or oppose xy and can't see their points get addressed/integrated or are polarized and manipulated so this could help a lot with all of that as well, e.g. to communicate why some policy is needed and why it's the way it is and not differently or to learn what the actual main issues with it are and why people would oppose xy. Ultimately I think it needs to be designed to be useful not just to interested experienced contributors but also (occasional unfamiliar) readers, just like one doesn't have to be able to or have edited Wikipedia to make regular use of it. [Here](https://www.kialo.com/will-kialo-ever-become-as-popular-as-the-largest-social-media-websites-63570) I listed ways Kialo and also this site could become (more) popular.

**Reply by keyserj** [reply]
> There also is another approach when it comes to high-level goals and use-uses besides such variables. That is particular use-cases.

Yeah, I think particular use-cases are good at helping people see realistic usage, at the cost of potentially limiting ideas of the possibilities. Understanding realistic usage is probably more important. But perhaps having both particular use-cases and variables would help accomplish both goals.

> I think a key one would be evidence-based effective policy as a system for merging and integrating opposing views and crowdsource / collective intelligence

That's a good one - I'm a huge fan of policymakers, citizens, and scientists all collaborating on the same Topic in order to result in a better-decisioned & happier society. I think there are probably a ton of enhancements (UX, not just performance) required in order to scale to 1000+ participants, but I do see it as possible.

> Also people are frustrated with politics or often can't communicate why they support or oppose xy and can't see their points get addressed/integrated or are polarized and manipulated so this could help a lot with all of that as well, e.g. to communicate why some policy is needed and why it's the way it is and not differently or to learn what the actual main issues with it are and why people would oppose xy.

:100: :100: :100: yes these political-related communication misunderstandings/struggles are a core motivation for Ameliorate.

> Ultimately I think it needs to be designed to be useful not just to interested experienced contributors but also (occasional unfamiliar) readers, just like one doesn't have to be able to or have edited Wikipedia to make regular use of it

Yeah I agree

> Here I listed ways Kialo and also this site could become (more) popular.

Nice, thanks for sharing.
